Documentation/Supported Features.md

```diff
@@ -61,20 +84,30 @@ This is the set of motions which are currently supported listed alphabetically.
 - cc
 - dd
 - D
+- ga
+- gd : using Goto.Definition from visual studio
```

Comment:
This is an alias for 'go to definition' most of the time, but in C/C++ it is actually (like vim) 'go to declaration'.  Also `gD` is supported.